storage/tianmu/exporter/data_exporter_txt.cpp

```diff
@@ -92,14 +92,14 @@ void DEforTxt::PutRowEnd() {
     WriteChars(line_terminator_);
 }
 
-size_t DEforTxt::WriteString(const types::BString &str, int len, bool text_or_bin) {
+size_t DEforTxt::WriteString(const types::BString &str, int len, bool text_or_bin, bool isNull) {
```

Comment:
Change `isNull` to `is_null`.
```
The names of variables (including function parameters) and data members are all lowercase, with underscores between words. See:
```
https://google.github.io/styleguide/cppguide.html#Variable_Names

Reply:
OK, I will fix it.